tests/test_fixtures.py

```diff
@@ -1,6 +1,6 @@
 #
 # Copyright (c) 2019 VMware, Inc. All Rights Reserved.
-# SPDX-License-Identifier: BSD-2-Clause
+# SPDX-pkg_license-Identifier: BSD-2-Clause
```

Comment:
Looks like a bulk replace :) . This change is not needed.

Reply:
yes my bad! will revert it back to License